deriv.com.conf

```diff
@@ -32,7 +32,26 @@ server {
     listen 80;
     server_name  _;
 
-    add_header Cache-Control "public, max-age=7200, s-maxage=600";
+    location ~* \.html$ {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location /page-data {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location = /sw.js {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location /static {
+        add_header Cache-Control "public, max-age=31536000, immutable";
```

Comment:
any reason why keep cache for 1 year?, isnt it too long, why dont put it a day or a week then it will be revalidated?

Reply:
These files get a unique hash from webpack (from content) so they are safe to cache forever: check https://www.gatsbyjs.org/docs/caching/

Reply:
nice!, 👍